pypd/models/alert.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) PagerDuty.
+# See LICENSE for details.
+try:
+    import ujson as json
+except ImportError:
+    import json
+
+from .entity import Entity
+
+
+class Alert(Entity):
+    def resolve(self, from_email=None):
+        """Resolve an alert using a valid email address."""
+        if from_email is None:
+            raise Exception('%s.resolve requires \'from_email\' argument.')
```

Comment:
Instead of just raising the `Exception` you could raise the `InvalidArguments` exception as mentioned below for the basic exceptions in this file :)